content/hardware/04.pro/boards/portenta-h7/tutorials/secure-boot/secure-boot.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Secure Boot on Portenta H7'
+description: 'Learn how to use secure boot on the Arduino Portenta H7.'
+tags: 
+  - Secure Boot
+author: 'Umberto Baldi'
+hardware:
+  - hardware/04.pro/boards/portenta-h7
+
+software:
+  - ide-v1
+  - ide-v2
+  - cli
+---
+
+## Introduction
+This short tutorial will guide the user through enabling the secure boot on the Portenta H7, generating custom security keys, and using them with the MCUboot bootloader.
+
+## Hardware & Software Required
+-   [Portenta H7](https://store.arduino.cc/portenta-h7)
+-   Arduino IDE 1.8.19+  or Arduino IDE 2.0.0-rc5+ (https://www.arduino.cc/en/software)
+-   [Arduino Mbed OS Portenta Boards](https://github.com/arduino/ArduinoCore-mbed) version 3.0.0+
+-   [imgtool](https://github.com/arduino/imgtool-packing/releases/latest) (optional)
+
+## Instructions
+
+### Flashing the Latest Bootloader
+In order to have secure boot enabled you must update the bootloader on your Portenta H7 and use [MCUboot](https://www.mcuboot.com/). You can find more info on how to perform the update in [this other tutorial](../por-ard-bl/content.md).
```

Comment:
```suggestion
In order to have secure boot enabled you must update the bootloader on your Portenta H7 and use [MCUboot](https://www.mcuboot.com/). You can find more info on how to perform the update in [this other tutorial](https://docs.arduino.cc/tutorials/portenta-h7/updating-the-bootloader).
```

Reply:
Better to use the doc root `path /tutorials/portenta-h7/updating-the-bootloaderà should we ever want to rename the subdomain